src/main/java/de/tum/in/www1/artemis/domain/Exercise.java

```diff
@@ -780,6 +783,14 @@ public void setSecondCorrectionEnabled(boolean secondCorrectionEnabled) {
         this.secondCorrectionEnabled = secondCorrectionEnabled;
     }
 
+    public boolean getFeedbackSuggestionsEnabled() {
+        return Boolean.TRUE.equals(feedbackSuggestionsEnabled);
```

Comment:
You are right 👍

Reply:
Update: We need support for `null` values here because existing exercises have `null` as the `feedback_suggestions_enabled` column. Maximilian and I decided to change it back here instead of migrating existing exercise rows.